packages/models/tests/unit/sql.spec.js

```diff
@@ -84,6 +84,14 @@ describe('normalize SQL', () => {
       normalize(`SELECT users.*, a.* FROM users WHERE a.user_id = 'alice'`)
     ).toEqual(`SELECT users.*, a.* FROM users WHERE a.user_id = ?`);
   });
+
+  test('removes comments correctly', () => {
+    expect(
+      normalize(
+        `SELECT 4 AS random; -- guaranteed random, generated by fair dice roll`
```

Comment:
Looks good! Do you also want to check that it does the right thing when the comment is embedded in the SQL string? And/or make sure it plays nicely with any other more complex SQL features?

Reply:
> Looks good! Do you also want to check that it does the right thing when the comment is embedded in the SQL string? 

Good point, it didn't. I've fixed it.

> And/or make sure it plays nicely with any other more complex SQL features?

Not sure what else to test. Note this normalization isn't meant to be fool-proof, nor can possibly be without being a full parser.

As our SQL parser becomes more complete it might make sense to remove this heuristic and instead normalize the AST and regenerate the SQL query from that.

Reply:
Ok. Also, I assume you tested this on the SQL that was provided by the user?